src/Movies/Movies.js

```diff
@@ -1,5 +1,5 @@
 import Movie from '../MovieCard/MovieCard';
-import SelectedMovie from '../SelectedMovie/SelectedMovie';
+import SelectedMovie from '../SelectedMovie/SelectedMovie.js';
```

Comment:
If adding the file extension here, be consistent and add the file extension on all the imports